app/models/dataservice/v1/process_external_activity_data_job.rb

```diff
@@ -9,7 +9,9 @@ def initialize(_learner, _content, _portal_start)
   end
 
   def lara_start
-    DateTime.parse(JSON.parse(content)['lara_start']) rescue Time.now()
+    # this will be nil if there aren't any dirty answers when lara sent the data
+    # and we want to leave it that way
+    DateTime.parse(lara_start_str) rescue nil
```

Comment:
@scytacki  I don't think `lara_start_str` is defined anywhere.  

Or at least I can't find it anywhere else when I check out this branch...

Reply:
Good catch!  
I'm looking at adding tests now. I should have done that before, and this error just proved why.